src/App.tsx

```diff
@@ -1,7 +1,70 @@
-import React from "react";
-
-function App() {
-  return <div className="App">Hey :)</div>;
-}
+import * as React from "react";
+import Map from "./containers/Map";
+import Sidebar from "./containers/Sidebar";
+import SidebarControl from "./containers/SidebarControl";
+import { Box } from "@material-ui/core";
+import { useWindowDimensions } from "./utils";
+import "./App.css";
+export const AppContext: any = React.createContext({});
 
+const App: React.FC = () => {
+  const { width } = useWindowDimensions();
+  const [openSidebar, setOpenSidebar] = React.useState(true);
```

Comment:
`const`'s at the beginning of the component would be nice to have them alphabetically